.github/workflows/build.yml

```diff
@@ -0,0 +1,43 @@
+name: "Build workflow"
+on:
+  pull_request:
+jobs:
+  build-test-windows:
+    runs-on: windows-2022
+    defaults:
+      run:
+        shell: msys2 {0}
+    steps:
+      - name: Install msys2
+        uses: msys2/setup-msys2@v2
+        with:
+          update: true
+          path-type: inherit
+      - name: Check out code
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Install golang
+        uses: actions/setup-go@v2
+        with:
+          go-version: '1.14.7'
+      - name: Install Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.7.9'
+      - name: Build Test
+        run: |
+          export ALGORAND_DEADLOCK=enable
+          export SKIP_GO_INSTALLATION=True
+          scripts/travis/build_test.sh
+      # - name: Upload artifact to Github
+      #   uses: actions/upload-artifact@v2
+      #   with:
+      #     if-no-files-found: error
+      #     path: $(git ls-files -o | grep -v crypto/libs | grep -v crypto/copies | grep -v crypto/libsodium-fork | grep -v crypto/lib/ | grep -v ^gen/ | grep -v swagger.json.validated | tr "\n" ":")
+      - name: Slack Notification
+        env:
+          SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
+        run: |
+          curl -X POST --data-urlencode "payload={\"text\": \"Nightly windows build test on Github failed. ${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }} \"}" $SLACK_WEBHOOK
+        if: ${{ failure() && contains(github.base_ref, 'rel/nightly') }}
```

Comment:
Can we get a message for `rel/*` here? Then, we can get failure messages for rel/beta merge too.

Reply:
Added